examples.lisp

```diff
@@ -14,6 +14,27 @@
     ((just x) x)
     (nothing else)))
 
+(defun bind-maybe (f m)
+  (match maybe m
+    ((just x) (funcall f x))
+    (nothing nothing)))
+
+(defun inc-maybe (x)
+  (just (1+ x)))
+
+(defun maybe->string (x)
+  (just (write-to-string x)))
+
+(string-equal "2" (do-notation bind-maybe
+		      ((sum (inc-maybe 1))
```

Comment:
can you get the indentation right here? maybe you have tabs on

Reply:
Emacs configuration issues 🤷‍♂️. Thanks.